server/emailAPI/IEmailAPI.ts

```diff
@@ -0,0 +1,14 @@
+import IEmailVerificationTemplate from "../serverAPI/model/emailVerification/IEmailVerificationTemplate";
+
+export default interface IEmailAPI {
+    /**
+     * Sends verification code to the email address.
```

Comment:
Pls update comment to describe timeout logic

Reply:
Email API is not responsible for timeout logic, controller is